.github/workflows/macos.yml

```diff
@@ -98,16 +98,15 @@ jobs:
 
       - name: Build (${{ matrix.build-type }})
         run: |
-          mkdir build
-          cd build
-          qmake -config ${{ matrix.build-type }} ../src/Notes.pro
-          make -j$(sysctl -n hw.logicalcpu)
+          export CMAKE_BUILD_PARALLEL_LEVEL=$(sysctl -n hw.logicalcpu)
+          cmake . -B build -DCMAKE_BUILD_TYPE=${{ matrix.build-type }}
+          cmake --build build
 
       - name: Build dmg (${{ matrix.build-type }})
         run: |
-          cd bin
+          cd build
           plutil -insert NSRequiresAquaSystemAppearance -bool true Notes.app/Contents/Info.plist
-          plutil -insert CFBundleName -string Notes Notes.app/Contents/Info.plist
+          plutil -replace CFBundleName -string Notes Notes.app/Contents/Info.plist
```

Comment:
I had to change this to `replace` instead of `insert` because it would complain that the "Notes" already existed. Do you have an idea why this suddenly happens?

Reply:
My guess is that when we build with `qmake`, that property didn't get created at all, but now with CMake it does.

CMake probably sets it based on the project name (i.e. `notes`, lowercase), so this change is fine because `Notes` is preferable, I think...